docs/general/dql/selects.rst

```diff
@@ -59,8 +65,7 @@ Aliases can be used to change the output name of the columns::
 
 The ``FROM`` clause is used to reference the relation this select query is
 based upon. Can be a single table, many tables, a view, a :ref:`JOIN <sql_joins>`
-or another ``SELECT`` statement. See :ref:`SELECT Reference
-<sql_reference_select>`
+or another ``SELECT`` statement.
```

Comment:
```suggestion
or another ref:`SELECT <sql_reference_select>` statement.
```

Reply:
fixed in https://github.com/crate/crate/pull/10866/commits/3be45f0cd09095a71f55099ba3aeb51b6fd3b48e